apps/web/test/lib/getSchedule.test.ts

```diff
@@ -785,6 +777,7 @@ describe("getSchedule", () => {
           {
             id: 1,
             slotInterval: 45,
+            length: 45,
```

Comment:
If it's possible to have `length` unset in production, we should have it the same way.
Maybe we can create both types of tests(where `length` is set and where it's not set)